tests/nested/manual/component-recovery-system/task.yaml

```diff
@@ -0,0 +1,158 @@
+summary: create a recovery system with a kernel module component and reboot into it
+
+details: |
+  This test creates a recovery system with a kernel module component and
+  validates that the newly created system can be rebooted into.
+
+systems: [ubuntu-24.04-64]
```

Comment:
Maybe this could be `[-ubuntu-1*, -ubuntu-20*, -ubuntu-22*]` instead so we make sure the test runs in future releases.